app/admin/counterparts.rb

```diff
@@ -0,0 +1,37 @@
+ActiveAdmin.register Counterpart do
+  menu false
+  actions :all, except: [:index, :show]
+  permit_params :description, :threshold, :stock, :project_id
+
+  form do |f|
+    f.inputs do
+      f.input :description
+      f.input :threshold
+      f.input :stock
+      f.input :project, as: :select
+    end
+    f.actions do
+      f.action :submit
+    end
+  end
+
+  controller do
+    def create
+      super { redirect_to_project_show and return if resource.valid? }
+    end
+
+    def update
+      super { redirect_to_project_show and return if resource.valid? }
+    end
+
+    def destroy
+      super { redirect_to_project_show and return if resource.valid? }
+    end
```

Comment:
La doc est pas forcément intuitive à trouver, mais AA utilise InheritedResources et tu peux faire un truc comme ça :

```ruby
super do |success, failure|
  success.html { redirect_to_project_show }
end
```
Cf : https://github.com/activeadmin/inherited_resources

Reply:
(`resource.valid?` ça dit pas si l'objet a été bien supprimé ou pas)

Reply:
(et `and` est interdit : https://github.com/rubocop-hq/ruby-style-guide#no-and-or-or)